aptos-move/aptos-vm/src/move_vm_ext/session.rs

```diff
@@ -106,6 +108,16 @@ where
             .into_change_set()
             .map_err(|e| e.finish(Location::Undefined))?;
 
+        // TODO: Once we are ready to connect aggregator with delta writes,
+        // make sure we pass them to the session output.
+        //
+        // Expeted changes will be:
+        //   * Use `Aggregator` for gas fees tracking in coin.
+        //   * Pass `aggregator_change_set` further to produce `DeltaChnageSet`.
+        //   * Have e2e tests and benchmarks.
+        let aggregator_context: NativeAggregatorContext = extensions.remove();
+        let _ = aggregator_context.into_change_set();
```

Comment:
It's a bit unusual to keep something like this in the code. Maybe comment those two lines out as well.